test/compilable/test17419.d

```diff
@@ -0,0 +1,29 @@
+// https://issues.dlang.org/show_bug.cgi?id=17419
+
+
+extern (C) int fooc();
+alias aliasc = fooc;
+
+static assert(__traits(getLinkage, fooc) == "C");
+static assert(__traits(getLinkage, aliasc) == "C");
+
+extern (C) int food();
+extern (C++) int foocpp();
+extern (Windows) int foow();
+extern (Pascal) int foop();
+extern (Objective-C) int fooobjc();
+extern (System) int foos();
+
+static assert(__traits(getLinkage, fooc) == "C");
```

Comment:
the symbol `food`, is not referenced in this test.
while `fooc` is referenced twice.